modules/actions/log.go

```diff
@@ -28,6 +29,9 @@ const (
 	defaultBufSize = MaxLineSize
 )
 
+// WriteLogs appends logs to DBFS file for temporary storage.
+// It doesn't respect the file format in the filename like ".zst", since it's impossibly appending to compressed files.
```

Comment:
>  since it's impossibly appending to compressed files.

I do not see why ....

The DBFS was designed to be (almost) fully compatible as a general filesystem, it implements Seeker/Reader/Writer.

So I think the zstd compression layer should be a general wrapper and it could work with DBFS.

```
caller -> zstd-seekable writer -> low-level filesystem (disk, dbfs, etc)
```

Reply:
You are right, but I forgot to explain another situation.

The logs from the runner are sent from multiple requests, and Gitea doesn't keep the file open to wait for the next requests; it just appends new lines, closes it, and reopens it next time.

So it should be "It's (almost) impossible to append new content to closed compressed files."

Reply:
e06fbdc820e586d4b321ea0ae3211a25b2fb76aa

So it's not related to DBFS; the point is that it's temporary.